content/en/boilerplates/experimental.md

```diff
@@ -0,0 +1,6 @@
+---
+---
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `experimental`.
```

Comment:
This change makes the alpha warning in [this doc](https://preliminary.istio.io/latest/docs/ops/configuration/traffic-management/dns-proxy/) ambiguous. We need a different word to describe `alpha` features now. Maybe:
```
This feature is intended for [early evaluation](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and considered `alpha`.
```

Reply:
I think that was wrong, so I'll update that doc and ping the people who created it.